src/js/components/Button/Button.js

```diff
@@ -243,7 +243,7 @@ const Button = forwardRef(
         </Box>
       );
     } else if (typeof children === 'function') {
-      contents = children({ hover, focus });
+      contents = children({ hover, focus, disabled });
```

Comment:
Can you add `focus` and `disabled` to the documentation for `children`?

Reply:
This change accidentally slipped in, its' not part of this PR. Thanks for catching it.